src/material-examples/cdk/clipboard/cdk-clipboard-overview/cdk-clipboard-overview-example.ts

```diff
@@ -0,0 +1,15 @@
+import {Component} from '@angular/core';
+
+/**
+ * @title Clipboard overview
+ */
+@Component({
+  selector: 'cdk-clipboard-overview-example',
+  templateUrl: 'cdk-clipboard-overview-example.html',
+  styleUrls: ['cdk-clipboard-overview-example.css'],
+})
+export class CdkClipboardOverviewExample {
+  value = 'Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ex fugiat recusandae ' +
+          'consectetur illoet alias. Quia, numquam alias. Modi dolorum hic perferendis ' +
+          'reiciendis doloribus officiis aliquid asperiores repellat deleniti rerum.';
```

Comment:
optional: I always like using text like this as an opportunity as something fun. Like, maybe part of the theme song to _Fresh Prince of Bel Air_